### PR TITLE
2025 Announcement Blogpost

### PR DESCRIPTION
**Pull request by rayegun**
Let's see how this looks. Comments please!

### CODE REVIEW

File: blog/2024/09/juliacon-2025-announcement.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# JuliaCon Global 2025 and JuliaCon 2024 Wrap-Up
+
+![Pittsburgh Skyline](/assets/blog/2024-JuliaCon/AdobeStock_50245829.jpeg)
+We’re happy to announce that JuliaCon Global 2025 will be held in Pittsburgh Pennsylvania, USA in the ==third week of July==! As we approach the conference next year we will make further announcements.
+
+A few important date estimates for you:
+
+- Call for Proposals will be open November 1 - January 31
```
Comment (kshyatt) [inline blog/2024/09/juliacon-2025-announcement.md:8]: Should add years to this, I think

File: blog/2024/09/juliacon-2025-announcement.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# JuliaCon Global 2025 and JuliaCon 2024 Wrap-Up
+
+![Pittsburgh Skyline](/assets/blog/2024-JuliaCon/AdobeStock_50245829.jpeg)
+We’re happy to announce that JuliaCon Global 2025 will be held in Pittsburgh Pennsylvania, USA in the ==third week of July==! As we approach the conference next year we will make further announcements.
```
Comment (kshyatt) [inline blog/2024/09/juliacon-2025-announcement.md:4]: comma needed between Pittsburgh and Pennsylvania

Comment (kshyatt) [reply blog/2024/09/juliacon-2025-announcement.md:4]: also can we put the actual numerical dates rather than "third week"

File: blog/2024/09/juliacon-2025-announcement.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+A few important date estimates for you:
+
+- Call for Proposals will be open November 1 - January 31
+- Volunteer, and scholarship applications will be open from December 1 - February 28
```
Comment (kshyatt) [inline blog/2024/09/juliacon-2025-announcement.md:9]: no comma between "Volunteer" and "and"

File: blog/2024/09/juliacon-2025-announcement.md (hunk 1 above)
Comment (kshyatt) [reply blog/2024/09/juliacon-2025-announcement.md:8]: Can we provide an estimate for when acceptances will be out?

Comment (kshyatt) [reply blog/2024/09/juliacon-2025-announcement.md:8]: Will we have a separate, earlier date for minis again this year?

File: blog/2024/09/juliacon-2025-announcement.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+As an alternative we are starting a new trial program for community members to propose videos for upload to the Julia YouTube Channel. If your proposal is accepted you’ll be asked to prepare your proposal into a full video for upload! This call for proposals will be evergreen, it will be open for as long as we run this program so you are welcome to propose a video at any time.
+
+Our YouTube channel is a fantastic forum for you to advertise your packages / work, present tutorials which drive users to the Julia ecosystem, or to present unique perspectives on your favorite problem in computer science, software development, or scientific computing! We have well over 100k views per month, and a lot of impressions on each video we upload.
+
+If this program interests you please fill out the following form:
+
+https://forms.gle/vHUsG4hbbVf5V472A
+
+### JuliaCon Locals
```
Comment (kshyatt) [inline blog/2024/09/juliacon-2025-announcement.md:32]: Or "JuliaCons Local"? 🤔

File: blog/2024/09/juliacon-2025-announcement.md (hunk 1 above)
Comment (ranjanan) [reply blog/2024/09/juliacon-2025-announcement.md:8]: I would say Jan 15 first, with the optional extension later. I think acceptances should be out by Feb end.

File: blog/2024/09/juliacon-2025-announcement.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+We had over a dozen community members reach out to talk about hosting a JuliaCon Local at JuliaCon 2024! 
+The first step in this process is always to start a local meetup. We’re happy to help with that as much as we can, you can reach out on the #community channel on Slack at any time. 
+
+Once you have an established meetup and you’d like to host a JuliaCon Local please reach out to raye@julialang.org or ranjan@julialang.org for more information.
+
+We are working on a unified playbook for how to build your local community, how to get access to awesome speakers, and how to find spaces and funds for your meetup, so please keep an eye out for that on our usual channels!
+
+## JuliaCon Global 2024 Wrap Up
+
+![JuliaCon Construction Edition](/assets/blog/2024-JuliaCon/AdobeStock_50245829.jpeg)
```
Comment (aviks) [inline blog/2024/09/juliacon-2025-announcement.md:43]: This is probably pointing to the wrong image?

File: blog/2024/09/juliacon-2025-announcement.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+\- Skylar Gering, CalTech: Diversity & Accessibility co-Chair
+
+\- Valentin Churavy, MIT: Diversity & Accessibility co-Chair
+
+\- Carsten Bauer: Proceedings Chair
+
+\- Avik Sengupta, JuliaHub: Organizer
+
+\- Ranjan Anantharaman, JuliaHub: Executive co-Chair
+
```
Comment (aviks) [inline blog/2024/09/juliacon-2025-announcement.md:94]: The tractor image is fun and all, but maybe end with a photo of the juliacon 2024 audience?

File: blog/2024/09/juliacon-2025-announcement.md (hunk 1 above)
Comment (rayegun) [reply blog/2024/09/juliacon-2025-announcement.md:8]: Can we get acceptances processed within January 22 - March 1?

File: blog/2024/09/juliacon-2025-announcement.md (hunk 4 above)
Comment (rayegun) [reply blog/2024/09/juliacon-2025-announcement.md:32]: Eh, as much as Attorneys General is fun I think it's kinda hard to parse. And the event as a whole is called JuliaCon Local

Is it more correct?

File: blog/2024/09/juliacon-2025-announcement.md (hunk 1 above)
Comment (lucaferranti) [reply blog/2024/09/juliacon-2025-announcement.md]: CfP this year closed on Feb 1st and notification of acceptance was sent on Feb 22nd, so doable I guess

Comment (rayegun) [reply blog/2024/09/juliacon-2025-announcement.md]: Okay sounds good I'll keep the dates currently listed unless y'all want to have a Program meeting briefly and discuss

Comment (lucaferranti) [reply blog/2024/09/juliacon-2025-announcement.md]: I left a more general comment in slack, but I think it's a bit orthogonal to the date themselves and does not prevent this from being published

**Comment by rayegun** [comment]
@ranjanan can you do a final check here and I'll post?

File: blog/2024/09/juliacon-2025-announcement.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+## Julia Community Efforts
+
+We have a couple new community efforts we’d like to talk about!
+
+### Julia Online Videos
+
+We have a lot of community members who want to give presentations at JuliaCon each year but can’t make it! We receive a lot of requests to submit videos to the conference, but for now we intend to run JuliaCon as an in-person conference first and foremost. Hybrid conferences take a lot of effort, and have a negative impact on the in-person experience.
+
+As an alternative, we are starting a new trial program called ***Julia Online*** for community members to propose videos for upload to the Julia YouTube Channel. If your proposal is accepted, you’ll be asked to prepare your proposal into a full video for upload! This call for proposals will be evergreen and will stay open for as long as we run this program, so you are welcome to propose a video at any time.
```
Comment (giordano) [inline blog/2024/09/juliacon-2025-announcement.md:28]: `***Julia Online***` is rendered as is, maybe you wanted bold and/or italic?

Comment (rayegun) [reply blog/2024/09/juliacon-2025-announcement.md:28]: Oh yes, how are bold and italics handled in this version of MarkDown?

Comment (giordano) [reply blog/2024/09/juliacon-2025-announcement.md:28]: I don't know for sure, but could try with
```suggestion
As an alternative, we are starting a new trial program called _**Julia Online**_ for community members to propose videos for upload to the Julia YouTube Channel. If your proposal is accepted, you’ll be asked to prepare your proposal into a full video for upload! This call for proposals will be evergreen and will stay open for as long as we run this program, so you are welcome to propose a video at any time.
```

Comment (giordano) [reply blog/2024/09/juliacon-2025-announcement.md]: Looks like it worked: https://julialang.netlify.app/previews/pr2145/blog/2024/09/juliacon-2025-announcement/#julia_online_videos

File: blog/2024/09/juliacon-2025-announcement.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+\- Valentin Churavy, MIT: Diversity & Accessibility co-Chair
+
+\- Carsten Bauer: Proceedings Chair
+
+\- Avik Sengupta, JuliaHub: Organizer
+
+\- Ranjan Anantharaman, JuliaHub: Executive co-Chair
+
+\– Raye Kimmerer, MIT: Executive co-Chair
```
Comment (giordano) [inline blog/2024/09/juliacon-2025-announcement.md:99]: Any reason for not using proper bullet lists throughout the post?

Comment (giordano) [reply blog/2024/09/juliacon-2025-announcement.md:99]: Also, if anything I presume this should be
```suggestion
\- Raye Kimmerer, MIT: Executive co-Chair
```

The difference is
```julia-repl
julia> '–'
'–': Unicode U+2013 (category Pd: Punctuation, dash)

julia> '-'
'-': ASCII/Unicode U+002D (category Pd: Punctuation, dash)
```

Comment (rayegun) [reply blog/2024/09/juliacon-2025-announcement.md]: Haha I think this is some weird artifact of copying from my Markdown editor into a different flavor? No idea why this would happen. Thanks for noticing

Comment (rayegun) [reply blog/2024/09/juliacon-2025-announcement.md]: I copied directly from Typora... Maybe I need to force it to use a different markdown flavor

File: blog/2024/09/juliacon-2025-announcement.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+* **Guillaume Dalle:** For his various contributions to the Julia ecosystem including DifferentiationInterface.jl and ModernJuliaWorkflows.
+
+Awardees are nominated by the Julia community, and then winners are selected by a committee of senior community members based on their holistic contributions across the ecosystem.
+
+### JuliaCon 2024 Organizers and Volunteers
+
+JuliaCon wouldn’t happen without an amazing group of organizers and volunteers. The organizing committee put in an incredible amount of work over the past year, and an incredible 70+ volunteers made the conference possible.
+
+I first want to highlight the incredible Local Organizing Committee. Led by Gareth Thomas, who is unmatched in his ability to bring people together for a common cause, the local committee was the key to both JuliaCon Local Eindhoven and JuliaCon Global 2024!
```
Comment (giordano) [inline blog/2024/09/juliacon-2025-announcement.md:65]: The rest of the article is written in plural style (also because the author is "JuliaCon Organizing Committee"), I guess this should also be plural
```suggestion
We first want to highlight the incredible Local Organizing Committee. Led by Gareth Thomas, who is unmatched in his ability to bring people together for a common cause, the local committee was the key to both JuliaCon Local Eindhoven and JuliaCon Global 2024!
```

**Comment by ranjanan** [comment]
Looks good to me. Let's post !
